<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8">
     <item>
      <widget class="QWidget" name="widgetSidebar" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
            <item>
             <widget class="QPushButton" name="btnBack">
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelBack">
              <property name="text">
               <string>Scan Config</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btnTests">
              <property name="text">
               <string>Tests</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnProxy">
              <property name="text">
               <string>Proxy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAuthentication">
              <property name="text">
               <string>Authentication</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnHTTP">
              <property name="text">
               <string>HTTP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCrawler">
              <property name="text">
               <string>Crawler</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnFormInputs">
              <property name="text">
               <string>Form Inputs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTestVectors">
              <property name="text">
               <string>Test Vectors</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTechnologies">
              <property name="text">
               <string>Technologies</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkAdvancedSettings">
              <property name="text">
               <string>Advanced Settings</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabTests">
        <attribute name="title">
         <string>Tests</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabProxy">
        <attribute name="title">
         <string>Proxy</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabAuthentication">
        <attribute name="title">
         <string>Authentication</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabHTTP">
        <attribute name="title">
         <string>HTTP</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabCrawler">
        <attribute name="title">
         <string>Crawler</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabFormInputs">
        <attribute name="title">
         <string>Form Inputs</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabTestVectors">
        <attribute name="title">
         <string>Test Vectors</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabTechnologies">
        <attribute name="title">
         <string>Technologies</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
